packages/core/src/sass/app/_common/components/account-common.scss

```diff
@@ -741,14 +767,13 @@
 
             .proof-of-identity__submit-button {
                 @include mobile-or-tablet-screen {
-                    margin: unset;
                     width: 100%;
-                    margin-inline: 1.6rem;
+                    // margin-inline: 1.6rem;
                 }
             }
 
             .form-body {
-                @include desktop-screen {
+                @include desktop {
```

Comment:
Why you are removing this newly added breakpoint? 👀

Reply:
it was a simple merge conflit its resolved 👍